src/resources/filters/quarto-pre/table-classes.lua

```diff
@@ -53,7 +53,7 @@ function table_classes()
       attr.classes = pandoc.List()
       tbl.caption.long[#tbl.caption.long] = pandoc.Plain(createTableCaption(caption_parsed, attr))
       if #quarto.utils.as_inlines(tbl.caption.long) == 0 then
-        tbl.caption.long = nil
+        tbl.caption.long = pandoc.Blocks({})
```

Comment:
@tarleb this is surprising to us. Do you know why setting to nil was not replacing the value ? 

`tbl.caption.long` was not modified 

This is related to pandoc 3.4 to 3.6.3 update

Reply:
I think it's kind of the same as when filter functions return `nil`. But I'll need to check.

Reply:
Is this new ?